test/tempfile_factory_test.rb

```diff
@@ -1,13 +1,13 @@
 require './test/helper'
 
 class Paperclip::TempfileFactoryTest < Test::Unit::TestCase
-  should "be able to generate a tempfile with the right name" do
-    file = subject.generate("omg.png")
-  end
-  should "be able to generate a tempfile with the right name with a tilde at the beginning" do
-    file = subject.generate("~omg.png")
-  end
-  should "be able to generate a tempfile with the right name with a tilde at the end" do
-    file = subject.generate("omg.png~")
+  should 'generate a Tempfile with a random basename' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.